src/execution/execution_common.cpp

```diff
@@ -23,11 +23,66 @@ auto GenerateSortKey(const Tuple &tuple, const std::vector<OrderBy> &order_bys,
  * You can ignore the remaining part of this file until P4.
  */
 
+/**
```

Comment:
Nit: there should be a header and copyright thing at the top of this file, and there should be an extra newline between the `execution/execution_common.h` files and everything else.

```
//===----------------------------------------------------------------------===//
//
//                         BusTub
//
// exeuction_common.cpp
//
// Identification: src/execution/execution_common.cpp
//
// Copyright (c) 2024-2024, Carnegie Mellon University Database Group
//
//===----------------------------------------------------------------------===//
```

Also, my LSP is telling me that `common/config.h`, `type/value.h` and `type/value_factory.h` are unused imports.